app/src/main/java/com/github/se/travelpouch/model/events/EventRepository.kt

```diff
@@ -53,27 +68,29 @@ class EventRepositoryFirebase(private val db: FirebaseFirestore) : EventReposito
         .get()
         .addOnSuccessListener { result ->
           val events = result?.mapNotNull { documentToEvent(it) } ?: emptyList()
-          onSuccess(events)
+          onSuccess(events.sortedByDescending { it.date })
         }
         .addOnFailureListener { e ->
           Log.e("EventRepository", "Error getting documents", e)
           onFailure(e)
         }
   }
 
-  /**
-   * This function adds an event to the collection of events in Firebase.
-   *
-   * @param event (Event) : the event we want to add on Firebase
-   * @param onSuccess (() -> Unit) : the function called when the event is correctly added to the
-   *   database
-   * @param onFailure ((Exception) -> Unit) : the function called when an error occurs during the
-   *   adding an event to the database
-   */
-  override fun addEvent(event: Event, onSuccess: () -> Unit, onFailure: (Exception) -> Unit) {
-    performFirestoreOperation(
-        db.collection(collectionPath).document(event.uid).set(event), onSuccess, onFailure)
-  }
+  //  /**
+  //   * This function adds an event to the collection of events in Firebase.
+  //   *
+  //   * @param event (Event) : the event we want to add on Firebase
+  //   * @param onSuccess (() -> Unit) : the function called when the event is correctly added to
+  // the
+  //   *   database
+  //   * @param onFailure ((Exception) -> Unit) : the function called when an error occurs during
+  // the
+  //   *   adding an event to the database
+  //   */
+  //  override fun addEvent(event: Event, onSuccess: () -> Unit, onFailure: (Exception) -> Unit) {
+  //    performFirestoreOperation(
+  //        db.collection(collectionPath).document(event.uid).set(event), onSuccess, onFailure)
+  //  }
```

Comment:
What is the purpose of these lines?

Reply:
Nothing. I tend to forget to do a cleanup. Sorry for the inconvenience